### ISSUE TITLE
Request for 1.17

### ISSUE THREAD

**Issue by russ1003** [KaptainWutax/SeedCracker#212]

**Comment by shortbow123** [comment]
From what I have seen this does work on 1.17, you just have to download the source code and build the plugin from that, there are instructions on how to do that on the readme file, after you install gradle of course. but you also need to put the fabric API in the mod folder as well as the jar file you compile. I have compiled this and gotten a world seed from it, I havent had time to see if it is 100% acurate but I am assuming it is.

**Comment by shinobu0224** [comment]
uh um i really am a newbie to this but can someone help me build the mod for 1.17 or atleast guide me through the process ?

**Comment by nixeoo** [comment]
> From what I have seen this does work on 1.17, you just have to download the source code and build the plugin from that, there are instructions on how to do that on the readme file, after you install gradle of course. but you also need to put the fabric API in the mod folder as well as the jar file you compile. I have compiled this and gotten a world seed from it, I havent had time to see if it is 100% acurate but I am assuming it is.

Do you think you can post the jar file somewhere? I can't get it to build I keep getting an error.

**Comment by xingadora** [comment]
me too

**Comment by MrStache7** [comment]
yeah I'm really new to this stuff if you could post a simple jar file to drag and drop into the mods folder that would be great because I have no idea how to build one on my own

**Comment by SylveonBottle** [comment]
Minecraft 1.17, compiled by myself. Feel free to delete this post if want people spoon-feeding.
[seedcracker-0.2.3-beta.zip](https://github.com/KaptainWutax/SeedCracker/files/6642669/seedcracker-0.2.3-beta.zip) (the mod .jar is inside of the .zip, github doesn't allow me to directly upload .jar files)
I'm just putting this here as proof that simply compiling directly from the github works with 1.17.
### I highly recommend that you build this project yourself instead of downloading a random .jar
There's no telling what kind of viruses I could've put in it. I could have a token logger in here, I could've put some password steallers, I could put RAT's or any other kind of nasty things.

I mean, I promise that I did nothing to it other than build directly from github clone. But what good is a promise of someone unrelated to the project?


You need to do the following to get it to work properly:
/seed finder category STRUCTURES ON
/seed finder category BIOMES ON
/seed finder category DECORATORS ON

I don't know which one of these categories helped me find the seed, but the default settings don't work on 1.17. I just turned everything on and I managed to get the seed.

Here's an image as proof that it's easy to get working.
![image](https://user-images.githubusercontent.com/17802843/121785240-3fb3d500-cb7e-11eb-92ec-45a8b74ca26c.png)

**Comment by MrStache7** [comment]
Thank's you're a legend!

**Comment by elwo-boop** [comment]
Thanks!
